coffee/lib/abstract-chosen.coffee

```diff
@@ -280,8 +281,19 @@ class AbstractChosen
     if /Android/i.test(window.navigator.userAgent)
       return false if /Mobile/i.test(window.navigator.userAgent)
     return true
+    
+  @tagname_is_valid: : (@form_field) ->
+  if @form_field.tagName.toUpperCase() != 'SELECT'
+    console.error 'Chosen error: invalid tagName ' + @form_field + ' is not a select'
```

Comment:
JQuery version  should be $(form_field).selector
don't know how to wrote it in coffee script